Code.gs

```diff
@@ -22,10 +22,10 @@
 */
 
 var sourceCalendars = [                // The ics/ical urls that you want to get events from along with their target calendars (list a new row for each mapping of ICS url to Google Calendar)
-                                       // For instance: ["https://www.calendarlabs.com/ical-calendar/ics/76/US_Holidays.ics", "US Holidays"]
-  ["icsUrl1", "targetCalendar1"],
-  ["icsUrl2", "targetCalendar2"],
-  ["icsUrl3", "targetCalendar1"]
+                                       // For instance: ["https://www.calendarlabs.com/ical-calendar/ics/76/US_Holidays.ics", "US Holidays",  "colorId 1 to 11: check colorIds.gs file"]
+  ["icsUrl1", "targetCalendar1", "11"],
```

Comment:
I really don't want to make this 3rd parameter required. That will be pretty confusing for new users.

Can we make it so a 3rd parameter to the calendar map (`[targetCalendar, icsURL, color]`) is optional rather than required? Then we could send -1 to `processEvent` if that 3rd parameter is unspecified (ie as the map is currently)

Reply:
Changed the docs examples to show it is optional.